numpy/core/tests/test_inf.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+from numpy import float64
+from numpy.testing import assert_equal
+
+class TestInf:
+    def test_inf_ufuncs(self):
+        """Test the various ufuncs"""
+
+        dividend = np.array(
+            [np.inf] * 5
+            + [-np.inf] * 5
+            + [324434.4354] * 2
+            + [-434354.45543] * 2, dtype=float64)
+        
+        divisor = np.array(
+            [np.inf, -np.inf, 8657463.3435, -59420603.453265, 0] * 2
+            + [np.inf, -np.inf] * 2, dtype=float64)
+        
+        expected = np.array(
+            [np.nan, np.nan, np.inf, -np.inf, np.nan,
+             np.nan, np.nan, -np.inf, np.inf, np.nan,
+             0, -0, -0, 0])
```

Comment:
This test doesn't work, equality doesn't check for signs and `-0` (the integer) evaluates to `0` anyway.

Reply:
i have updated the test with a custom method to check for zeroes with sign

Reply:
The ci builds seems to be failing in the pipeline, although all tests pass locally and I didn't change any build file. May i check if the issue is on my side?